README.md

```diff
@@ -282,7 +156,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] MIT license link throws a 404-page request error when visited. Kindly endeavor to fix it.

**Broken License link:**
![image](https://user-images.githubusercontent.com/12745474/230933647-779ca9d5-7099-4901-ad5f-6fee4fd821c3.png)

Reply:
Fixed.